servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java

```diff
@@ -2938,9 +2784,6 @@ public InstanceIdentity postInstanceRefreshInformation(ResourceContext ctx, Stri
             String domain, String service, String instanceId, InstanceRefreshInformation info) {
 
         final String caller = "postinstancerefreshinformation";
-        final String callerTiming = "postinstancerefreshinformation_timing";
-
-        metric.increment(HTTP_POST);
```

Comment:
postInstanceRefreshInformation has many metrics that are not related to the HTTP request (such as "providerconfirm"). I didn't change them.